federation/backends/collections.md

```diff
@@ -1,6 +1,27 @@
 # Collections
 This page describes requirements on STAC collection metadata for backend providers in the openEO platform federation. These requirements should be considered an addition to what is already required by the [openEO API](https://api.openeo.org/#tag/EO-Data-Discovery/operation/describe-collection).
 
+## Collection availability
+
+The collections are a key asset of the platform, and users need to know what they can expect from a certain collection.
+The federation relies on openEO collection metadata following the STAC specification to communicate this towards the user. Backend providers
+need to comply with these requirements for their collection:
+
+The 'experimental' flag from https://stac-extensions.github.io/version/v1.1.0/schema.json needs to be set on experimental collections. 
+A collection can be experimental either because there are issues with the actual data or catalog, or because of backend specific issues that make the use of the collection unstable. 
+A collection marked as experimental does not need to comply with further requirements. 
+
+Collections need to indicate the key 'providers' that are responsible for ensuring access to the data and continuity in the case of active missions. The user may depend on the guarantees offered by these
+providers with respect to the properties (timeliness, completeness,...) of a specific collection. 
+The providers with role 'host' and 'producer' are mandatory.
```

Comment:
@jdries Can you explain a bit more why we need to add host? We currently only have 'processor' in our collections (which is in our case is the same as the 'host'). We currently also don't list producer so if we merge this, we will need a couple of days to update our collections.

Reply:
According to STAC, the host is where the underlying data is stored. When a user wants to determine how reliable a collection is, the reputation of the host is crucial. In most cases, it is impossible for a single openEO backend to offer more reliability than the host of the data.

Reply:
PS: I don't think it's a problem if we need a few days to update.

Reply:
I see. I thought of the host as the entity owning the S3 bucket/paying for the storage. In that case host for our data would be in most cases AWS so your statement about the reliability is very much true. I don't see a problem with adding this.